src/System.Console/tests/project.lock.json

```diff
@@ -424,11 +406,11 @@
           "lib/DNXCore50/System.Threading.ThreadPool.dll": {}
         }
       },
-      "xunit/2.1.0": {
+      "xunit/2.1.0-beta3-build3029": {
```

Comment:
XUnit.NetCore.Extensions most recent version depended on higher version of XUnit.NetCore than the one used by XUnit package, which was causing a build failure. So, as work-around I manually bumped the netcore extension version. This surfaces only after my change, since project.json file is updated which forces an update to project.lock.json